spec/models/admin/api_submission_settings_spec.rb

```diff
@@ -19,58 +19,8 @@
       end
     end
 
-    context 'service_output_json_key' do
-      let(:deployment_environment) { 'dev' }
-      let(:service_output_json_endpoint) { 'http://this-valid-url.com' }
-      let(:params) { { deployment_environment:, service_output_json_endpoint:, service_output_json_key: } }
-      let(:expected_error) { described_class::KEY_LENGTH_ERROR }
-
-      context 'has to be 16 characters long' do
-        let(:service_output_json_key) { '98957af1a0424376' }
-
-        it 'is valid' do
-          expect(subject).to be_valid
-        end
-      end
-
-      context 'is empty to reset configuration' do
```

Comment:
This test was removed but I suggested adding a validation for presence - it is probably worth adding a test that covers the new behaviour, 

''   = now not valid
nil = now not valid

Could also be that my suggestion was bad and '' should be valid, in which case I'd add that test and remove the `validates presence` for the key

Reply:
I tested the case we remove the config because it's the way to delete but I can add a unit test covering that. Thank you.